src/osx/cocoa/dataview.mm

```diff
@@ -1653,8 +1499,6 @@ -(id) init
         currentlyEditedColumn =
             currentlyEditedRow = -1;
 
-        [self setAutoresizesOutlineColumn:NO];
```

Comment:
I think this line was removed accidentally, so I'm going to restore it, please let me know if I'm wrong.